tests/test_arr_venue.py

```diff
@@ -756,3 +756,45 @@ def test_setup_matching(self, venue, client, helpers, request_page, selenium):
         assert url
         assert 'Reviewers/-/Assignment' in url.get_attribute('href')
 
+        # Desk Reject Paper 2
+        desk_reject_note = openreview.Note(
+            invitation='aclweb.org/ACL/ARR/2021/September/Paper2/-/Desk_Reject',
+            forum=submissions[3].forum,
+            replyto=submissions[3].forum,
+            readers=['aclweb.org/ACL/ARR/2021/September',
+                     'aclweb.org/ACL/ARR/2021/September/Paper2/Authors',
+                     'aclweb.org/ACL/ARR/2021/September/Paper2/Reviewers',
+                     'aclweb.org/ACL/ARR/2021/September/Paper2/Area_Chairs',
+                     'aclweb.org/ACL/ARR/2021/September/Program_Chairs'],
+            writers=[venue.get_id(), venue.get_program_chairs_id()],
+            signatures=[venue.get_program_chairs_id()],
+            content={
+                'desk_reject_comments': 'PC has decided to reject this submission.',
+                'title': 'Submission Desk Rejected by Program Chairs'
+            }
+        )
+
+        posted_note = pc_client.post_note(desk_reject_note)
+        assert posted_note
+
+        helpers.await_queue()
+
+        ## Reviewer reviewer_arr2@mit.edu tries to accept invitation for desk-rejected submission but gets an error
+        invite_edges = pc_client.get_edges(invitation='aclweb.org/ACL/ARR/2021/September/Reviewers/-/Invite_Assignment',
```

Comment:
it is strange that edge can be posted if the note is already desk rejected.

Reply:
edge was already posted before the desk-reject, it's just getting the posted edge here.

Reply:
You are right 🤦‍♀️